python/ray/tune/trial.py

```diff
@@ -308,6 +309,22 @@ def __init__(self,
         self._state_json = None
         self._state_valid = False
 
+    def _setup_resources(self, log_always: bool = False):
+        if not self.placement_group_factory and \
+           not int(os.getenv("TUNE_PLACEMENT_GROUP_AUTO_DISABLED", "0")):
+            try:
+                self.placement_group_factory = resource_dict_to_pg_factory(
+                    self.resources)
+            except ValueError as exc:
+                if log_always or log_once("tune_pg_extra_resources"):
+                    logger.warning(exc)
+                self.placement_group_factory = None
```

Comment:
can you log a version deprecation warning (i.e., extra_cpu/extra_gpu will be deprecated in a later version of Ray?)